src/types/service_development_config.go

```diff
@@ -4,4 +4,5 @@ package types
 type ServiceDevelopmentConfig struct {
 	Dependencies []DevelopmentDependencyConfig
 	Scripts      map[string]string `yaml:",omitempty"`
+	Ports        []string          `yaml:",omitempty"`
```

Comment:
We want to keep this as a single port for now. We will be getting to multiple ports soon enough.

Reply:
But I think we get this for free, no? All we're doing is creating another entry in the `ports` block of the yml file which docker-compose handles for us

Reply:
For multiple ports @alexdavid abnd I had taled about:
```
ports: 
  key1: 3000
  key2: 4000
```
where the keys will be used in the env variables we pass to other services

Reply:
Ah, ok. How are we managing env vars for ports right now? Is it ok to just have `port: 80` in service.yml?

Reply:
We aren't doing the env vars for ports yet (that will be the next task). Yes, for now please update this so you would add `port: 80` (separate ports under dev / production